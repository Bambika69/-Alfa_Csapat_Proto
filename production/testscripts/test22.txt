createfield
createvirologist F1
creategeneticcode paralyze -v V1
startturn V1
craftagent virus paralyze
