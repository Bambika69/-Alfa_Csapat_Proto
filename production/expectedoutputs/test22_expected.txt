Field created:
ID: F1
Type: Field
Neighbours: null

Virologist created:
ID: V1
Field: F1
Backpack: B1

GeneticCode created:
ID: GC1
Type: paralyze
Destination: V1

V1's turn started.

Agent crafted:
ID:null
Type: virus
GeneticCode: paralyze
Result: Failed
